Makefile

```diff
@@ -27,13 +27,11 @@ start-s3rver:
 test: test-js test-python
 test-js: test-prairielearn test-prairielib test-grader-host test-packages
 test-prairielearn: start-support
-	@nyc --reporter=lcov mocha --full-trace tests/index.js
+	@mocha --full-trace --parallel --require tests/mocha-hooks.mjs "tests/**/*.test.{js,mjs}"
```

Comment:
Maybe we should add a target to run the tests serially, and document this in the dev docs? I can imagine that there will be some cases where it will be much easier to debug a failing test if we can run it serially.

Also, do we need any updates to the docs about how to run individual tests? See: https://github.com/PrairieLearn/PrairieLearn/blob/2add41190efddfe4eae142634cef6ef57de862a3/docs/installingLocal.md?plain=1#L49

Reply:
Nice!